Composer/packages/client/src/components/GetStarted/GetStartedNextSteps.tsx

```diff
@@ -0,0 +1,310 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useEffect, useState, useMemo } from 'react';
+import { useRecoilValue } from 'recoil';
+import formatMessage from 'format-message';
+import { TeachingBubble } from 'office-ui-fabric-react/lib/TeachingBubble';
+import { ScrollablePane } from 'office-ui-fabric-react/lib/ScrollablePane';
+
+import TelemetryClient from '../../telemetry/TelemetryClient';
+import { localBotsDataSelector } from '../../recoilModel/selectors/project';
+import { currentProjectIdState } from '../../recoilModel';
+import { ManageLuis } from '../ManageLuis/ManageLuis';
+import { ManageQNA } from '../ManageQNA/ManageQNA';
+import { dispatcherState, settingsState } from '../../recoilModel';
+import { mergePropertiesManagedByRootBot } from '../../recoilModel/dispatchers/utils/project';
+import { rootBotProjectIdSelector } from '../../recoilModel/selectors/project';
+import { navigateTo } from '../../utils/navigation';
+
+import { GetStartedTask } from './GetStartedTask';
+import { h3Style } from './styles';
+
+type GetStartedProps = {
```

Comment:
you can avoid redefining these types below if you move them to a common types file

Reply:
I will do that, but didn't intend to redefine. they just got moved.